helpers.go

```diff
@@ -54,6 +54,26 @@ func RawExec(ctx context.Context, driver Driver, query string, dest ...interface
 	return nil
 }
 
+// RawExecArgs will execute given query with given arguments.
+// If an object is given, it will mutate it to match the row values.
+func RawExecArgs(ctx context.Context, driver Driver, query string, args []interface{}, dest ...interface{}) error {
+	if driver.hasLogger() {
+		start := time.Now()
+		query := NewRawQuery(query)
```

Comment:
```go
query := Query{
	Raw:   query,
	Query: query,
	Args:  args,
}
```

Reply:
args is not a map[string]interface{}

Reply:
I like less and less this new function 😕

Reply:
I think we can change `Args` to a `[]interface{}` since we use that here:

https://github.com/ulule/makroud/blob/master/helpers.go#L27

And I would like to removes `sqlx` dependencies in the long term.

Reply:
@thoas @gillesfabio @yansal @edouardparis What do you think ?

Reply:
let's change it in this PR